Resources/Prototypes/Entities/Objects/Devices/Syndicate_Gadgets/reinforcement_teleporter.yml

```diff
@@ -25,7 +25,7 @@
   - type: GhostRole
     name: ghost-role-information-syndicate-reinforcement-spy-name
```

Comment:
this one is kinda wrong as well, since the radio is used for all three types, not only the spy

Reply:
I'm not to sure what to do about this one. I didn't even touch this one it was just like this. Looking at the ghost roles with each one they all state the correct thing so it I think it's fine?

Reply:
The name set here is for the default role that the radio creates. Spy is correct